modules/autoskill.lua

```diff
@@ -148,9 +149,16 @@ executeSkill = function ()
 		local isFirstSkill = 1
 		if currentSkill ~= '0' and currentSkill ~= '#' then
 			for command in string.gmatch(currentSkill, ".") do
+				-- Check that skills can be clicked
+				while not _castingNP and not BATTLE_REGION:exists(GeneralImagePath .. "battle.png" ) do end

```

Comment:
I think `_castingNP` can be replaced for `_battle.hasClickedAttack()`.
And the infinite loop can be replaced for a `wait("blablabla.png", 10)`, where 10 is a timeout. Maybe just 5 seconds is enough?